solr/CHANGES.txt

```diff
@@ -237,6 +237,9 @@ led to the suppression of exceptions. (Andrey Bozhko)
 
 * SOLR-17545: Upgrade to Gradle 8.10 (Houston Putman)
 
+* SOLR-17556: Examples should run with the standard & recommended Solr home and solr data dir (except for
```

Comment:
[0] Minor nit - the "should" here might confuse users.  Is this how Solr behaves in 9.8?  Is it how it should behave? etc.

Wording like `"home" and "data" directories used by Solr examples have been updated to align with documented best practices.` might be a little clearer.